lib/lotus/generators/action.rb

```diff
@@ -108,6 +114,14 @@ def assert_action!
         end
       end
 
+      # @since x.x.x
+      # @api private
+      def assert_http_method!
+        if !HttpRouter::Route::VALID_HTTP_VERBS.include?(_http_method.upcase)
```

Comment:
Lotus::Routes currently defines the DSL. Not sure if it should go there.

I'm unsure why those methods are defined at all. Can not find anything that references that list of methods at all (besides my PR). What is the purpose?